elections/tools/generate_electorate.py

```diff
@@ -0,0 +1,134 @@
+#!/usr/bin/env python3
+
```

Comment:
You probably want to add an [SPDX license header](https://github.com/kata-containers/documentation/blob/master/Licensing-strategy.md) in here - the rest of the project uses SPDX style and is predominantly apache2.